WTWSMS/history/characters/hebrew.txt

```diff
@@ -11,7 +11,7 @@
 	}
 }
 
-# IDs above 10 digits cause problems
+# IDs above 10 digits may cause problems
```

Comment:
We know they cause problems, the most notable of which are crashes.

Reply:
But not all 10 digit character cause problems IIRC. Only characters with IDs above integer limit (2147483647) should be problematic.

Reply:
I would prefer to not risk it...

Reply:
I don't think there's any risk in this case :)